### DISCUSSION TITLE
The term 'Set-PnPUnifiedgroup' is not recognized as a name of a cmdlet

### DISCUSSION THREAD

**Discussion by Fringie** [pnp/powershell#3582] [General] [answered]
Hello,

Goal:
I am trying to use the Set-PnPUnifiedGroup cmdlet to disable the welcome message when adding users to Teams (Group Sites).

Issue:
When I try to use the cmdlet 'Set-PnPUnifiedgroup' I get the below error:
![image](https://github.com/pnp/powershell/assets/7203124/e8e042fb-1906-4c5a-9276-7adb50566326)
As you can see in the above screenshot, I am using PNP PowerShell 2.2 which is supposed to have the cmdlet. I am able to use non M365 group cmdlets such as get-pnpweb or add-pnplistitem but the PNPUnifiedGroup cmdlets don't seem to be there. Any ideas?

**Comment by AndersRask** [comment, marked as answer]
I think they removed the old alias that was a remnant from an older repository.

Try substituting PnPUnifiedgroup with PnPMicrosoft365Group for all related cmdlets

**Reply by Fringie** [reply]
Ah that was the issue. Unfortunatly the cmdlet you mentioned doesn't have the parameter I need but I ended up using New-PNPMicrosoft365Group instead which ended up working for me.

Thanks for the help, it sent me in the right direction when I was going in circles.
